X/Xorg_libXt/bundled/patches/Xtos.h.patch

```diff
@@ -0,0 +1,10 @@
+--- a/include/X11/Xtos.h
++++ b/include/X11/Xtos.h
+@@ -49,6 +49,6 @@
+     defined(__s390x__) || \
+     (defined(__hppa__) && defined(__LP64__)) || \
+     defined(__amd64__) || defined(amd64) || \
+-    defined(__powerpc64__))
++    defined(__powerpc64__)
```

Comment:
How did this even work

Reply:
Beats me. Maybe the C preprocessor doesn't parse `#elif` conditions if that branch isn't taken?

Reply:
I found https://gitlab.freedesktop.org/xorg/lib/libxt/-/merge_requests/72#note_2718431, they claim it doesn't affect GCC>=4.6 nor Clang>=3.0, but that seems false since we _were_ affected.

Reply:
I think we might be hit because we're cross-compiling, or because we're compiling for "strange" architectures. The original code has an `#if` statement before this condition that should trigger most of the time, but apparently it doesn't trigger for us, and we hit this faulty code.